Makefile

```diff
@@ -1,7 +1,13 @@
-__PHONY__: build build-testing
+__PHONY__: build build-testing build-dev
 
 build:
 	docker build -t stellar/quickstart -f Dockerfile .
 
 build-testing:
-	docker build -t stellar/quickstart:testing -f Dockerfile.testing .
\ No newline at end of file
+	docker build -t stellar/quickstart:testing -f Dockerfile.testing .
+
+build-dev:
+	# Buildkit is disabled because of https://github.com/moby/buildkit/issues/2463.
+	DOCKER_BUILDKIT=0 docker build -t stellar/stellar-core:testing -f docker/Dockerfile.testing git://github.com/stellar/stellar-core#master --build-arg CFLAGS='' --build-arg CXXFLAGS='-stdlib=libc++' --build-arg CONFIGURE_FLAGS='--disable-tests'
```

Comment:
It appears building from git contexts has a bug (see https://github.com/moby/buildkit/issues/2463) and so we need to disable buildkit for this build. The env var makes sure that if someone has buildkit enabled this will disable it for this run at least.

Reply:
I found out how to make the build work with buildkit by passing `--build-arg BUILDKIT_CONTEXT_KEEP_GIT_DIR=true` to the builder.